### PR TITLE
Updated exercise for 8.3 to showcase a focus on deployment

### PR DESCRIPTION
**Pull request by matmille**

### CODE REVIEW

File: content/workshops/ansible_automation/exercise1.0.adoc [hunk 1]
```diff
@@ -9,17 +9,66 @@ layout: lab
...
+[NOTE]
+We have configured 'sudo' for you; however, Ansible can use lots of other methods to become (-b) the superuser. 
+====
+
+[source,bash]
+----
+ansible localhost -m uri -a "url=https://raw.githubusercontent.com/RedHatGov/redhatgov.workshops/master/ansible_tower_aws/files/code-server.service return_content=yes dest=/etc/systemd/system" -b
+----
+
+Now in the */etc/systemd/system* directory lives the *code-server.service* file you downloaded from the SCM system. In order to apply it, we would typically utilize the "systemctl" command; however, this is an ansible workshop.  With that being said, we will apply the configuration file with a module.
```
Comment (Tekwriter1221) [inline content/workshops/ansible_automation/exercise1.0.adoc:42]: Capitalize Ansible

File: content/workshops/ansible_automation/exercise1.0.adoc [hunk 2]
```diff
@@ -9,17 +9,66 @@ layout: lab
...
+
+It is possible now to see your service running by doing the following commands:
+
+[source,bash]
+----
+systemctl status code-server.service
+----
 
-=== Set Up Dependencies:
+Additionally, you can more details about pod deployment:
```
Comment (Tekwriter1221) [inline content/workshops/ansible_automation/exercise1.0.adoc:56]: A word missing? Additionally, you can *add* more pod deployment details. (?)

**Comment by matmille** [comment]
Updated syntax

File: content/workshops/ansible_automation/exercise1.0.adoc (hunk 1 above)
Comment (Tekwriter1221) [reply content/workshops/ansible_automation/exercise1.0.adoc]: however, this is an ansible workshop. Capitalize Ansible

**Pull request merged**